.github/workflows/license-check.yml

```diff
@@ -2,7 +2,7 @@ name: License Check
 on:
   pull_request:
     paths:
-      - 'yarn.lock'
+      - 'pnpm.lock'
```

Comment:
I already wrote a big review about similar mistakes here https://github.com/graphql/graphiql/pull/3054 so I will not repeat

Reply:
my local is not the same as the commit you are reviewing, there must be a mistake